pydis_site/templates/wiki/delete.html

```diff
@@ -0,0 +1,90 @@
+{% extends "wiki/base.html" %}
+{% load wiki_tags sekizai_tags static %}
+
+
+{% block wiki_pagetitle %}Delete Article{% endblock %}
+
+{% block wiki_contents %}
+  <h1 class="title is-1">Delete "{{ article.current_revision.title }}"</h1>
+
+  {% if cannot_delete_root %}
+    <article class="message is-warning">
+      <div class="message-body">
+        <p>
+          You cannot delete the root article.
+        </p>
+
+        <a href="{% url 'wiki:get' path=urlpath.path article_id=article.id %}" class="button is-primary">
+          <span class="icon">
+            <i class="fas fa-arrow-left"></i>
+          </span>
+          <span>Go back</span>
+        </a>
+      </div>
+    </article>
+  {% else %}
+    {% if cannot_delete_children %}
```

Comment:
shouldn't these checks go over django's permission system or how does this work?

Reply:
They do. That's a wiki thing.